### DISCUSSION TITLE
What happened to the API documentation?

### DISCUSSION THREAD

**Discussion by pelly** [observablehq/plot#1618] [Q&A] [answered]
It's been a few months since I looked at Plot. Previously, I was able to find some extensive, detailed API level documentation. Is that still available? If so, where? All I can currently find is the helpful (but not always detailed enough for me) [new guide](https://observablehq.com/plot/)

Thanks,
John

**Comment by Fil** [comment, marked as answer]
The new documentation entirely subsumes and replaces the older README.md (here's the [last version](https://github.com/observablehq/plot/blob/a5308ae6ec9051d1b48e867305782d3391d41aff/README.md), for reference).

We did a lot of editing and rewriting (a book’s worth of content…) to try and bring more context and clarity to every section. However, unless we made a mistake, no details were removed. Each page of the new site now begins with a gentle description of the feature, and ends with the complete reference that used to be on the README page. If you find anything that has gone missing, please report a bug.

What's different is that you can't search the whole page with browser search any more—but we feel it is advantageously replaced by vitepress’ instant search (ctrl-K).
